charts/argo-cd/templates/argocd-notifications/service-metrics.yaml

```
@@ -2,9 +2,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "argo-cd.notifications.fullname" . }}-metrics
+  name: {{ template "argo-cd.notifications.fullname" . }}
```

My mistake, let's keep the `-metrics` on the serivice, sorry for that
```suggestion
  name: {{ template "argo-cd.notifications.fullname" . }}-metrics
```

---

yeah, i was on the fence as well and think it looks better with the `-metrics` suffix